test/fuzz/datetime_strptime_fuzzer.cc

```diff
@@ -0,0 +1,31 @@
+extern "C"
```

Comment:
Usually this should not be necessary, and it's up-to header itself to guard his definitions with something like:
```
#ifdef __cplusplus
extern "C" {
#endif
```

I wonder, which of those headers was unprotected and led to compile errors?

Reply:
already removed